spring-06/library/src/main/java/ru/otus/testing/commandline/BookCommands.java

```diff
@@ -0,0 +1,73 @@
+package ru.otus.testing.commandline;
+
+import jakarta.validation.constraints.NotNull;
+import org.springframework.shell.standard.ShellComponent;
+import org.springframework.shell.standard.ShellMethod;
+import ru.otus.testing.model.Author;
+import ru.otus.testing.model.Comment;
+import ru.otus.testing.model.Genre;
+import ru.otus.testing.service.BookService;
+import ru.otus.testing.service.CommentService;
+import ru.otus.testing.service.PrintBookService;
+
+@ShellComponent
+public class BookCommands {
+
+    private final BookService bookService;
+
+    private final CommentService commentService;
+
+    private final PrintBookService printBookService;
+
+    public BookCommands(BookService bookService, CommentService commentService, PrintBookService printBookService) {
+        this.bookService = bookService;
+        this.commentService = commentService;
+        this.printBookService = printBookService;
+    }
+
+    @ShellMethod(value = "readById-book", key = {"readById-book", "-rbi-book"})
+    public String readBookById(long bookId) {
+        var presentedBookInfo = bookService.findById(bookId);
+        return printBookService.printBookToConsole(presentedBookInfo);
+    }
+
+    @ShellMethod(value = "readAll-book", key = {"readAll-book", "-rall-book"})
```

Comment:
shell:>readAll-book
failed to lazily initialize a collection of role: ru.otus.testing.model.Book.comments: could not initialize proxy - no Session